custom/conf/app.example.ini

```diff
@@ -827,6 +827,9 @@ LEVEL = Info
 ;; Dependencies can be added from any repository where the user is granted access or only from the current repository depending on this setting.
 ;ALLOW_CROSS_REPOSITORY_DEPENDENCIES = true
 ;;
+;; Enable OpenStreetMap button in Location field on user profiles
+;ENABLE_OSM_BUTTON = true
```

Comment:
Since Gitea is an open source project for everyone, I think this "location service provider" should be able to be customized, like `http://my-map/?loc={loc}`

Reply:
Customization is indeed good and all, and I considered creating a list of different providers:
- Apple Maps: Can't do that.
- Google Maps: I don't believe I can do that.
- [OpenRailwayMap](https://wiki.openstreetmap.org/wiki/OpenRailwayMap) and some other OpenStreetMap derivatives (I didn't try all of them): Searching using a URL query is not possible.

I am not sure about the extend to which this would be useful.

Reply:
* `https://www.google.com/maps/place/405 W 45th St, New York, NY 10017, USA/`
* `https://www.mapquest.com/search/405 W 45th St, New York, NY 10017, USA`
* `https://www.bing.com/maps?where1=Microsoft Way, Redmond, WA`
etc

Reply:
I tried to address this but I am now being met with the following error:

```
2023/07/29 12:03:16 ...s/graceful/server.go:70:NewServer() [I] Starting new Web server: tcp:0.0.0.0:3000 on PID: 
2023/07/29 12:03:17 .../context_response.go:86:HTML() [E] Render failed: failed to render template: user/profile, error: template error: builtin(static):shared/user/profile_big_avatar:32:29 : executing "shared/user/profile_big_avatar" at <.ContextUser.LocationQueryURL>: can't evaluate field LocationQueryURL in type interface {}
----------------------------------------------------------------------
						<a href="{{.ContextUser.LocationQueryURL | Safe}}{{.ContextUser.Location}}" rel="nofollow noreferrer">
						                      ^
----------------------------------------------------------------------
```

Reply:
I forgot to push the changes I was talking about and I also have no idea why I used `.ContextUser`, has been fixed.